### PR TITLE
Mobile View Info Panel

### PR DESCRIPTION
**Pull request by npfitz**
EPIC #22826 

Handles showing the info panels while using the mobile view. Info panel appears from the right side of the screen

chrome_4qk7hUztsh

### CODE REVIEW

**Review by ranquild — APPROVED** [review]

File: frontend/src/metabase/dashboard/components/DashboardInfoSidebar.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const DashboardInfoSidebar = ({
   );
 
   return (
-    <DashboardInfoSidebarRoot>
+    <DashboardInfoSidebarRoot data-testid="sidebar-right">
```
Comment (kulyk) [inline frontend/src/metabase/dashboard/components/DashboardInfoSidebar.tsx:72]: Can we make `DashboardInfoSidebarRoot` an HTML `aside` to keep HTML more semantic?

**Review by kulyk — APPROVED** [review]
LGTM, just a small request

**Review by WiNloSt — APPROVED** [review]
Looks a lot better now 🎉

File: frontend/src/metabase/dashboard/components/DashboardInfoSidebar.tsx (hunk 1 above)
Comment (npfitz) [reply frontend/src/metabase/dashboard/components/DashboardInfoSidebar.tsx:72]: Good idea. It was originally wrapped in an aside but no reason we can't just change it. Will do!

**Pull request merged**